IoTuring/Configurator/MenuPreset.py

```diff
@@ -169,7 +173,7 @@ def AddEntry(self,
                  display_if_key_value={},
                  instruction="",
                  question_type="text",
-                 choices=[]) -> None:
+                 choices: List[ChoiceDict]=[]) -> None:
```

Comment:
```suggestion
                 choices: list[ChoiceDict]=[]) -> None:
```

If you use lowercase l you don't need to import `typing.List`

Reply:
If i read the Update Notes correctly, this is only possible since python 3.9.
Our pyproject.toml states =>3.8. So we could get conflicts with people using 3.8

Reply:
Isn't that lowercase list is part of  `__future__.annotations`? So it works without anything in 3.9, in 3.8 we need `__future__`

I mean we have this in multiple files and tests never fail on 3.8:
- https://github.com/richibrics/IoTuring/blob/9c07015094fdeb38d0175608c4256a90e9eff96e/IoTuring/ClassManager/ClassManager.py#L34
- https://github.com/richibrics/IoTuring/blob/9c07015094fdeb38d0175608c4256a90e9eff96e/IoTuring/Settings/SettingsManager.py#L17

Also Python 3.8 is EOL now, we can drop suport for it if it's easier.